std_msgs/Int16 bin_num
std_msgs/Float64MultiArray bin_object_list
std_msgs/Float64MultiArray transformation_old_new
sensor_msgs/PointCloud2 input_rgb_cloud
---
geometry_msgs/Point centroid
geometry_msgs/Point normal
geometry_msgs/Point obj_left
geometry_msgs/Point obj_top
geometry_msgs/Point obj_right
